### ISSUE TITLE
Button: icon slot doesn't receive correct classes from button preset's icon field. 

### ISSUE THREAD

**Issue by NbNewbanking** [primefaces/primevue#5877]
### Describe the bug

It appears to me that the icon slot in Button component doesn't work as expected, therefore limiting the possibilities of a developer when using pass-through for creating a customized UILibrary out of PrimeVue components.

It is clearly visible when comparing to Tag component. When I use Tag's icons slot, and I access the props sent to this slot, I can see the classes coming from it's preset being passed to the slot's props. It allows me to easily use custom SVG icons inside this slot.

Identical approach with Button component does not result in such behaviour of the icon slot. No matter what I do, I always receive an empty string. For testing purposes I even passed the "iconClass" prop to the Button component, and even this doesn't make the slot contain the classes neither from preset nor from the "iconClass" prop. I suspect the slot to be a bit broken, due to it's inability of receiving neither pass-through nor props-sent classes. 

### Reproducer

https://stackblitz.com/edit/guzv8h-ar9zw6?file=src%2FApp.vue

### PrimeVue version

3.35.0

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vite

### Browser(s)

Chrome 125

### Steps to reproduce the behavior

Open the stackblitz link and look into the console.
The console is blank, and slotProps of Button component's icon slot has an empty string assigned to it's 'class' property.
Then add "icon" prop to the Button component on line 2. It can be any value.
The console is still blank, and slotProps of Button component's icon slot has an empty string assigned to it's 'class' property - even though the icon prop is used.

### Expected behavior

After adding the icon prop, or even after just adding the icon slot, the slotProps' 'class' property should contain at least the 'test-class' specified in the button pass-through preset for 'icon' pass-through property. 

In short, it should act the same way as Tag component does - compute the classes for icon slot and pass them through slotProps.

**Comment by benoitkopp** [comment]
I have the same issue on the Message component, when passing a class to the `icon` attribute in the passthrough, I'm getting an empty string when using the props of the icon's slot.

**Comment by tugcekucukoglu** [comment]
The fix is already in place on our side and will be included in the upcoming version 5 release. Closing for now.
